src/main/java/com/codahale/metrics/riemann/JvmMetricSet.java

```diff
@@ -1,118 +0,0 @@
-package com.codahale.metrics.riemann;
```

Comment:
Delete - this is using metrics2 to report in metrics3, which is part of what I want to remove (the duplicate dependency)